Source/Scene/Scene.js

```diff
@@ -764,15 +768,23 @@ define([
         // initial guess at frustums.
         var near = camera.frustum.near;
         var far = camera.frustum.far;
-        var numFrustums = Math.ceil(Math.log(far / near) / Math.log(this.farToNearRatio));
-        updateFrustums(near, far, this.farToNearRatio, numFrustums, this._frustumCommandsList, false, undefined);
+        var farToNearRatio;
+
+        if (this._logDepthBuffer) {
+            farToNearRatio = this.farToNearRatio * this.farToNearRatio;
```

Comment:
Is it better to expose a separate `logarithmDepthFarToNearRatio`?  Or to expose a `logarithmDepthFarToNearRatioScale` that is multiplied?

Just squaring it seems ad hoc.